setup.py

```diff
@@ -5,7 +5,7 @@
 
 
 install_requires = (
-    'djangorestframework>=3.1.0,<3.2',
+    'djangorestframework>=3.2.0,<3.3',
```

Comment:
Not that I know of, I just wanted it to use 3.2 for tests/local installs.  Do I have to faff around to maintain 3.1 support as well?